compute_endpoint/globus_compute_endpoint/engines/high_throughput/interchange.py

```diff
@@ -977,8 +977,10 @@ def start(self, poll_period: int | None = None) -> None:
 
                                 mdata["tasks"][task_container].remove(tid)
 
-                                # Transfer any outstanding task statuses to the
-                                # result message
+                                # Yadu: Todo needs a fix
+                                # Each Result is now an independent message, there's no
+                                # piggybacking of status deltas
+                                """
                                 if tid in self.task_status_deltas:
                                     r["task_statuses"] += self.task_status_deltas[tid]
                                     del self.task_status_deltas[tid]
```

Comment:
I'm going to approve this PR for now, but this is an issue in `unfork_htex_3` I think, and we should work it through.  Let's please not forget about this.

Reply:
@khk-globus These bits are addressed now with the story in the git message.